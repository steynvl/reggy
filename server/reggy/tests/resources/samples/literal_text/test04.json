{
  "sampleStringsInfo": [
    [
      {
        "markerType": "Literal text",
        "markedStrings": [
          "jannis"
        ],
        "markerInfo": {
          "caseInsensitive": false,
          "matchAllExceptSpecified": false
        },
        "repeatInfo": {
          "repeat": "1"
        }
      }
    ],
    [
      {
        "markerType": "Literal text",
        "markedStrings": [
          "kannis"
        ],
        "markerInfo": {
          "caseInsensitive": false,
          "matchAllExceptSpecified": false
        },
        "repeatInfo": {
          "repeat": "Custom range",
          "start": 3,
          "end": 7
        }
      }
    ],
    [
      {
        "markerType": "Literal text",
        "markedStrings": [
          "\""
        ],
        "markerInfo": {
          "caseInsensitive": false,
          "matchAllExceptSpecified": false
        },
        "repeatInfo": {
          "repeat": "1"
        }
      }
    ]
  ],
  "generalRegexInfo": {
    "startRegexMatchAt": "Anywhere",
    "endRegexMatchAt": "Anywhere",
    "regexTarget": "Java"
  },
  "generateMethod": "samplesAndSemantics"
}
