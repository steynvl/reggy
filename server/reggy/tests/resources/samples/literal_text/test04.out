Java
Pattern regex = Pattern.compile("jannis(?:kannis){3,7}\"");
jannis(?:kannis){3,7}\"
Perl
my $regex = /jannis(?:kannis){3,7}"/;
jannis(?:kannis){3,7}"
POSIX
jannis(?:kannis){3,7}"
jannis(?:kannis){3,7}"
Python
regex = re.compile(r'jannis(?:kannis){3,7}"')
jannis(?:kannis){3,7}"
JavaScript
const regex = /jannis(?:kannis){3,7}"/;
jannis(?:kannis){3,7}"
PHP
$regex = '/jannis(?:kannis){3,7}"/';
jannis(?:kannis){3,7}"
Golang
regex, _ := regexp.Compile("jannis(?:kannis){3,7}\"")
jannis(?:kannis){3,7}\"
Rust
let re = Regex::new(r"jannis(?:kannis){3,7}\"").unwrap();
jannis(?:kannis){3,7}\"
C#
var re = new Regex(@"jannis(?:kannis){3,7}\"");
jannis(?:kannis){3,7}\"
Scala
val re = "jannis(?:kannis){3,7}\"".r
jannis(?:kannis){3,7}\"
Kotlin
val regex = Regex("jannis(?:kannis){3,7}\"")
jannis(?:kannis){3,7}\"